### ISSUE TITLE
Source 0.020 clear PETG

### ISSUE THREAD

**Issue by simonwex** [simonwex/faceshield#5]

**Comment by simonwex** [comment]
Plasticworld.ca
Talking to: James
416-630-6745

20 4'x8' sheets @ $20 a sheet
$80 shipping

$586 including tax and shipping

**Comment by simonwex** [comment]
https://www.johnstonplastics.com/edmonton/
780-465-0431
Talking to Ryan

$22.60
0-10

$19 per sheet
11 - 49

**Comment by simonwex** [comment]
![image](https://user-images.githubusercontent.com/25260/77365097-bca52580-6d12-11ea-8eea-d8ebf273de34.png)

Ordered a box of GBC00158. I think this'll work and be a very easy thing to use.

**Comment by whomisi** [comment]
Good work Simon, is this something your CNC could cut ? Or worst case, I guess you could CNC a template or two

**Comment by revzimmer** [comment]
Signs ink laser would be able to cut it I bet

**Comment by whomisi** [comment]
yeah but it's pretty small, it's a Legend Helix: 24" x 18" (610 x 457 mm)

**Comment by revzimmer** [comment]
Cut a bunch at once??
